projects/plugins/jetpack/modules/google-fonts/class-jetpack-google-fonts.php

```diff
@@ -0,0 +1,193 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Module: Jetpack Google Fonts
+ *
+ * @package automattic/jetpack
+ */
+class JetpackGoogleFonts {
```

Comment:
Maybe we could avoid the phpcs issue with a different naming?

```suggestion
<?php
/**
 * Utilities to enable our Google Fonts when the module is enabled.
 *
 * @package automattic/jetpack
 */

namespace Automattic\Jetpack;

/**
 * Register Jetpack Google Fonts and add support for it.
 */
class Google_Fonts {
```